spec/requests/targets/index._spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+describe 'POST /targets', type: :request do
+
+  let!(:user)  { create(:user) }
+  let!(:target) { create(:target) }
+  let!(:targets) { create_list(:target, 3, user: user) }
+
+  subject { get targets_path, headers: auth_user_headers, as: :json }
+
+  it 'returns a successful response' do
+    subject
+    expect(response).to be_successful
+  end
+
+  it 'returns a list with 3 targets' do
+    subject
+    expect(json[:targets].count).to eq 3
+  end
+
+  it 'returns the correct information' do
+    subject
+    expect(json[:targets]).to include_json(
+      targets.map do |target|
+        {
+          id: target.id,
+          topic_id: target.topic_id,
+          title: target.title,
+          radius: target.radius,
+          latitude: target.latitude.round(10),
+          longitude: target.longitude.round(10)
+        }
+      end
+    )
+  end
```

Comment:
maybe this needs a context like ```context 'when the user gets his targets' do ```

Reply:
Yeah or maybe 'when getting the user\'s targets'

and avoid using the `his`

Reply:
Like, a cultural thing?